README.md

```diff
@@ -225,6 +229,37 @@ Then, assuming you've installed the Hayhooks package in a virtual environment, y
 pip install trafilatura
 ```
 
+## Run pipelines from the CLI
+
+### Run a pipeline from the CLI JSON-compatible parameters
+
+You can run a pipeline by using the `hayhooks pipeline run` command. Under the hood, this will call the `run_api` method of the pipeline wrapper, passing parameters as the JSON body of the request.
+This is convenient when you want to do a test run of the deployed pipeline from the CLI without having to write any code.
+
+To run a pipeline from the CLI, you can use the following command:
+
+```shell
+hayhooks pipeline run <pipeline_name> --param 'question="is this recipe vegan?"'
+```
+
+### Run a pipeline from the CLI uploading files
```

Comment:
1. I think that the file upload is a big feature and it deserves its own section on README, maybe linking to the `rag_indexing_query` example folder.
2. The current section (Run a pipeline from the CLI uploading files) might be a bit misleading if we don't explicitly mention that the underlying pipeline wrapper must have a specific signature to support file uploads. If we add the dedicated file upload section (point 1), we can reference it here to provide better clarity.

Reply:
Agree on 1 and 2. Will add a link to the example and mention about `run_api` signature!